Pod/Classes/XExtensionItem.h

```diff
@@ -102,6 +103,22 @@
                             attachments:(NSArray/*<NSItemProvider>*/ *)attachments NS_DESIGNATED_INITIALIZER;
 
 /**
+ Initialize a new instance with placeholder data and a type identifier which will be used by the activity controller to 
+ determine which activities and extensions are displayed, and an array of attachments that will actually be passed to 
+ the selected activity/extension.
+ 
+ @param placeholderData    A placeholder item. System activities that do not accept `NSExtensionItem` input
+ will receive this placeholder instead.
+ @param dataTypeIdentifier The UTI for the data.
+ @param attachments        An array of media data associated with the extension item. These items must be of
+ 
+ @return New item source instance.
+ */
+- (instancetype)initWithPlaceholderData:(id)placeholderData
+                     dataTypeIdentifier:(NSString *)dataTypeIdentifier
```

Comment:
If the object can be useful with some subset of parameters, make those required and assert. If all of them are required, make them all required.